h2o-k8s/tests/clustering/h2o-service.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: h2o-deployment # Deployment used for better job observability during tests, do not use in production.
+  namespace: default
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: h2o-k8s
+  template:
+    metadata:
+      labels:
+        app: h2o-k8s
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: h2o-k8s
+          image: 'harbor.h2o.ai/opsh2oai/h2o-3-k8s-runner:1'
```

Comment:
Should the image version number be hardcoded or parametrized?

Reply:
We would have to propagate it to the docker container. Which is definitely possible. And the sed it in here.